commit fe24d15d8645b60a372feb5b1f4c47576691d9d4
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-04-23T15:16:06-07:00 (6 months ago)
    
    TRITON-1628 migration provision test failing with "invalid networks" when no fabrics available
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/test/vms.migrate.test.js b/test/vms.migrate.test.js
index 75f8813..f688f0d 100644
--- a/test/vms.migrate.test.js
+++ b/test/vms.migrate.test.js
@@ -167,7 +167,7 @@ exports.get_admin_fabric_network = function (t) {
 
             t.ok(networks[0], 'Admin fabric network should be found');
             if (Array.isArray(networks) && networks.length >= 1) {
-                PROVISION_NETWORKS = {uuid: networks[0].uuid};
+                PROVISION_NETWORKS = [{uuid: networks[0].uuid}];
             }
 
             t.done();
@@ -188,7 +188,7 @@ exports.config_setup = function configSetup(t) {
             assert.string(cfg.packageName, 'cfg.packageName');
             assert.object(cfg.vm, 'cfg.vm');
 
-            cfg.vm.networks = [PROVISION_NETWORKS];
+            cfg.vm.networks = PROVISION_NETWORKS;
 
             vasync.pipeline({funcs: [
                 function loadPackages(_, next) {
